### PR TITLE
Updating the correct description of the property

### PR DESCRIPTION
**Pull request by nikmahes**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`npm run build`) was run locally and any changes were pushed
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by liamdebeasi** [comment]
Hey there! Can you run `npm run build`?

**Comment by nikmahes** [comment]
Sure, Let me check !

File: core/src/components/fab-list/readme.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default Example
 
 | Property    | Attribute   | Description                                                         | Type                                    | Default    |
 | ----------- | ----------- | ------------------------------------------------------------------- | --------------------------------------- | ---------- |
-| `activated` | `activated` | If `true`, the fab list will be show all fab buttons in the list.   | `boolean`                               | `false`    |
+| `activated` | `activated` | If `true`, the fab list will show all fab buttons in the list.   | `boolean`                               | `false`    |
```
Comment (brandyscarney) [inline core/src/components/fab-list/readme.md]:
```suggestion
| `activated` | `activated` | If `true`, the fab list will show all fab buttons in the list.      | `boolean`                               | `false`    |
```

Comment (brandyscarney) [reply core/src/components/fab-list/readme.md]: This needs to be updated to pass the build

**Comment by brandyscarney** [comment]
Thank you!

**Pull request merged**